samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/public/auth-start.html

```diff
@@ -0,0 +1,48 @@
+<!--This file is used during the Teams authentication flow to assist with retrieval of the access token.-->
+<!--If you're not familiar with this, do not alter or remove this file from your project.-->
+<html>
+<head>
+    <title>Login Start Page</title>
+    <meta charset="utf-8" />
+</head>
+
+<body>
+    <script src="https://res.cdn.office.net/teams-js/2.7.1/js/MicrosoftTeams.min.js" integrity="sha384-4Gy2G+qxzDVdrdemcVqKVQvaSK1Ghg3x6xcsaMLPc/pw7KPtiogHGM97LTWF2PWg" crossorigin="anonymous"></script>
+    <script type="text/javascript"
+            src="https://alcdn.msauth.net/browser/2.21.0/js/msal-browser.min.js"
+            integrity="sha384-s/NxjjAgw1QgpDhOlVjTceLl4axrp5nqpUbCPOEQy1PqbFit9On6uw2XmEF1eq0s"
+            crossorigin="anonymous">
+    </script>
+    <script type="text/javascript">
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.getContext().then(async (context) => {
+                // Generate random state string and store it, so we can verify it in the callback
+                var currentURL = new URL(window.location);
+                var clientId = currentURL.searchParams.get("clientId");
```

Comment:
where are we passing client ID

Reply:
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/969174d4-c68a-46e8-bdc1-a72d684422d9)